src/InstrumentationEngine.Lib/StringUtils.h

```diff
@@ -5,6 +5,11 @@
 
 #include <string>
 
+#ifndef E_BOUNDS
+// Apparantly, Linux builds don't have E_BOUNDS defined.
+#define E_BOUNDS 0x8000000BL
+#endif
```

Comment:
This is in unix.h now.

Reply:
It is, however this is going directly to the release branch which doesn't have the change (yet)

Reply:
I'll move it to unix.h so it matches main